### PR TITLE
added Big Inning and miscellaneous improvements

### PR DESCRIPTION
**Pull request by tonywagner**
- featured videos, including Big Inning
- audio-only streams
- unique game icons
- probable pitchers
- doubleheader/TBD start times
- restored "play all recaps" behavior when selecting date
- moved more text to language localization file
- added AddonIsEnabled detection for inputstream adaptive
- code cleanup

### CODE REVIEW

**Comment by eracknaphobia** [comment]
Thanks! I will take a look.

File: resources/lib/globals.py [hunk 1]
```diff
@@ -261,15 +258,14 @@ def addPlaylist(name,game_day,mode,icon,fanart=None):
     if fanart is None: fanart = FANART
     liz.setArt({'icon': icon, 'thumb': icon, 'fanart': fanart})
     liz.setInfo( type="Video", infoLabels={ "Title": name } )
-    info = {'plot': LOCAL_STRING(30375)+game_day,'tvshowtitle':'MLB','title':name,'originaltitle':name,'aired':game_day,'genre':LOCAL_STRING(700),'mediatype':'video'}
+    game_day_display = parse(game_day).strftime('%A, %B %-d, %Y')
```
Comment (eracknaphobia) [inline resources/lib/globals.py:261]: @tonywagner I was getting an error on this line.
```
game_day_display = parse(game_day).strftime('%A, %B %-d, %Y')
ValueError: Invalid format string
```
Removing the `-` from `%-d` seemed to fix it. Can you verify?

**Review by eracknaphobia — COMMENTED** [review]
Just had the issue with game_day_display but other than that everything else looks great!. This is a nice update to the addon!

File: resources/lib/globals.py (hunk 1 above)
Comment (tonywagner) [reply resources/lib/globals.py:261]: Yes, that fixes it. Good catch! I just pushed another commit to my fork for this, if that helps.

(Apparently "%-d", which drops the leading zero, is only a Mac/Linux thing?)

**Pull request merged**